src/codegen/jsonschema/additionalproperties.rs

```diff
@@ -30,10 +30,10 @@ pub fn from_object_with_additional_properties(
                     model: Box::new(model?),
                 })))
             }
-            Value::Bool(_) => Err(Error::SchemaInvalidProperty(
-                // todo: bool support maybe as a flag
-                "additionalProperties".to_string(),
-            )),
+            Value::Bool(true) => Ok(Model::new(ModelType::MapType(MapType {
+                name: Some(name),
+                model: Box::new(AnyType::model(schema, scope).flatten(container, scope)?),
+            }))),
             _ => Err(Error::SchemaInvalidProperty(
                 "additionalProperties".to_string(),
             )),
```

Comment:
i think we also need to change 41 line as default is true for additionalProperties